src/algorithms/acquisition/adapters/beidou_b1c_pcps_acquisition.cc

```diff
@@ -255,12 +255,12 @@ void BeidouB1cPcpsAcquisition::set_local_code()
     // Perform acquisition in Pilot signal
```

Comment:
You need to enable acq_iq_. I think the error you mentioned was fixed in our emails by applying the ICD equation when combining and doing addition

Reply:
Yes, I enabled "acq_iq_".